app/api/users.py

```
@@ -0,0 +1,200 @@
+    
+    query = select(User).offset(skip).limit(limit)
+    result = await db.execute(query)
+    users = result.scalars().all()
+    
+    return [UserResponse.model_validate(user) for user in users]
+
+
+@router.get("/{user_id}", response_model=UserResponse)
+async def get_user(
+    user_id: str,
+    db: AsyncSession = Depends(get_db)
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)